app/src/main/java/org/fossasia/openevent/app/module/event/list/EventsListAdapter.java

```diff
@@ -77,6 +78,7 @@ public int getItemCount() {
         public void bind(Event event) {
             this.event = event;
             binding.setEvent(event);
+            binding.setSelectedEventId(ContextManager.getSelectedEvent() == null ? -1 : ContextManager.getSelectedEvent().id);
```

Comment:
This is calling getSelectedEvent each time the layout is bound... twice...

You can move this in constructor